packages/desktop-lib/src/lib/desktop-os-inactivity-handler.ts

```
@@ -125,52 +185,85 @@ export class DesktopOsInactivityHandler {
+		this._powerManager.detectInactivity.removeListener(
+			'activity-proof-result-accepted',
+			this._activityProofResultAcceptedHandler
+		);
+	}
 
-		if (offlineMode.enabled) {
-			this._powerManager.window.webContents.send('update_view', params);
+	private cleanUpDialog(): void {
+		if (this._session.dialog) {
+			this._session.dialog.close();
+			delete this._session.dialog;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)